### ISSUE TITLE
Add harvester for DataFirst, University of Cape Town

### ISSUE THREAD

**Issue by erinspace** [fabianvf/scrapi#325]
http://www.datafirst.uct.ac.za/

**Comment by fabianvf** [comment]
Oh, cool

**Comment by fabianvf** [comment]
Do they have any sort of API?

**Comment by fabianvf** [comment]
Think I figured out a way...

**Comment by erinspace** [comment]
@fabianvf progress on this?

**Comment by fabianvf** [comment]
no, it ended up being more difficult than anticipated to integrate, and the service went down for a little while, so I put it on hold while I worked on other priorities. The pull request is still up, if you'd like to look at it.
